templates/options-checkboxes.php

```diff
@@ -1,15 +1,24 @@
-<?php $cb_id = $this->get_element_id() . '-' . esc_attr( sanitize_text_field( $data_row['value'] ) ); ?>
+<?php
+/**
+ * Default template for Fieldmanager_Checkboxes.
+ *
+ * @package Fieldmanager
+ */
+
+$cb_id = $this->get_element_id() . '-' . sanitize_text_field( $data_row['value'] );
+?>
+
 <div class="fm-option">
 	<input
 		class="fm-element"
 		type="checkbox"
 		value="<?php echo esc_attr( $data_row['value'] ); ?>"
 		name="<?php echo esc_attr( $this->get_form_name( '[]' ) ); ?>"
 		id="<?php echo esc_attr( $cb_id ); ?>"
-		<?php echo $this->get_element_attributes(); ?>
-		<?php echo $this->option_selected( $data_row['value'], $value, "checked" ); ?>
+		<?php echo $this->get_element_attributes(); // Escaped interally. xss ok. ?>
+		<?php echo esc_attr( $this->option_selected( $data_row['value'], $value, 'checked' ) ); ?>
```

Comment:
While I admit I didn't consider that possibility, as far as I can tell this returns just `checked`, not `checked="checked"`.